airgap/terraform/main.tf

```diff
@@ -231,6 +231,11 @@ resource "aws_instance" "lh_registry_aws_instance" {
     aws_security_group.lh_registry_aws_secgrp.id
   ]
 
+  root_block_device {
+    delete_on_termination = true
+    volume_size = 40
```

Comment:
curious: what's the default value if not specified?

Reply:
default value is 8 GiB